packages/editor/src/browser/editor-preferences.ts

```diff
@@ -355,7 +408,7 @@ export interface EditorConfiguration {
     'editor.roundedSelection'?: boolean
     'editor.minimap.enabled'?: boolean,
     'editor.minimap.showSlider'?: string,
-    'editor.minimap.renderCharacters'?: boolean,
+    'editor.minimap.renderCharacters'?: boolean
```

Comment:
some props have comma at the end of line, some don't. could you make it consistent please ?